src/main/scala/dsptools/DspTester.scala

```diff
@@ -238,12 +242,22 @@ class DspTester[T <: Module](c: T,
     }
   }
 
+  def nearlyEqual(a: Double, b: Double): Boolean = {
+    val epsilon = 1e-12
+    val divisor: Double = if(b == 0.0) b.abs else epsilon
+    val result = (a - b).abs / (divisor + epsilon) < epsilon
```

Comment:
I don't think the `+ epsilon` is needed if the previous line ensures divisor =/= 0